x/update_validators/errors.go

```diff
@@ -0,0 +1,35 @@
+package update_validators
+
+import (
+	"fmt"
+
+	"github.com/confio/weave"
+	"github.com/confio/weave/errors"
+	"github.com/confio/weave/x"
+)
+
+// ABCI Response Codes
+// x/update_validators reserves 40 ~ 49.
+const (
+	CodeEmptyDiff    uint32 = 40
+	CodeWrongType           = 41
+	CodeNoPermission        = 42
+)
+
+var (
+	errEmptyDiff    = fmt.Errorf("Empty validator diff")
+	errWrongType    = fmt.Errorf("Wrong type for accounts storage")
+	errUnauthorized = fmt.Errorf("Not authorized to perform this transation")
+)
+
+func ErrEmptyDiff() error {
+	return errors.WithLog("", errEmptyDiff, CodeEmptyDiff)
```

Comment:
If you don't add any dynamic info, just use `errors.WithCode(errEmptyDiff, CodeEmptyDiff)` WithLog is there to add dynamic data but still allow `IsXYZErr()` checks to work